src/styles.scss

```diff
@@ -1 +1,44 @@
 /* You can add global styles to this file, and also import other style files */
+
```

Comment:
add normalization styles (which reset user-agent style rules)

Also I am not sure we need to setup our own styles for form components on this stage. Much better to get default styles from the UI library we are gonna use